package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@bigbinary/neeto-hotkeys",
+  "version": "1.0.0",
+  "description": "React hook for handling keyboard shortcuts, offering a customizable and platform-aware solution, supporting global and scoped modes.",
+  "scripts": {
+    "prepare": "husky install"
+  },
+  "engines": {
+    "node": ">=18.12"
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,json}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "{package.json,.eslintrc.js,.prettierrc.js}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    ".eslint-rules/**/*": [
+      "prettier --write",
+      "eslint --fix"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bigbinary/neeto-hotkeys.git"
+  },
+  "author": "BigBinary",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/bigbinary/neeto-hotkeys/issues"
+  },
+  "homepage": "https://github.com/bigbinary/neeto-hotkeys#readme",
+  "devDependencies": {
+    "@babel/eslint-parser": "^7.23.3",
+    "@babel/preset-env": "^7.23.3",
+    "@babel/preset-react": "^7.23.3",
+    "@bigbinary/eslint-plugin-neeto": "^1.1.31",
+    "@bigbinary/neeto-cist": "^1.0.4",
```

Comment:
Why is this not added to peerDepenendencies?

Reply:
Added neeto-cist to peerDependencies.